Tests/js/views/assets/ez-actionbarview-tests.js

```diff
@@ -37,49 +37,49 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             this.view.destroy();
         },
 
-        "Should instantiate createContentActionView with contentType": function () {
-            var createContentActionFound = false;
+        _isActionCreated: function (actionId, testedAttributes) {
+            var actionFound = false,
+                that = this;
 
             for (var i = 0; i < this.view.get('actionsList').length; i++){
-                if (this.view.get('actionsList')[i].get('actionId') === 'createContent'){
-                    createContentActionFound = true;
-                    Assert.areSame(
-                        this.view.get('contentType'),
-                        this.view.get('actionsList')[i].get('contentType'),
-                        'The contentType should have been set to createContentActionView'
-                    );
+                if (this.view.get('actionsList')[i].get('actionId') === actionId){
+                    actionFound = true;
+
+                    /*jshint loopfunc: true */
```

Comment:
Oh right, I have separated iteration through actions and iteration through attributes.